src/coreclr/src/jit/hwintrinsicarm64.cpp

```diff
@@ -236,7 +240,22 @@ bool HWIntrinsicInfo::isInImmRange(NamedIntrinsic id, int ival, int simdSize, va
 {
     assert(HWIntrinsicInfo::lookupCategory(id) == HW_Category_IMM);
 
-    return ival <= lookupImmUpperBound(id, simdSize, baseType) && ival >= 0;
+    int immLowerBound = 0;
+
+    switch (id)
+    {
+        case NI_AdvSimd_ShiftLeftLogicalAndInsert:
+        case NI_AdvSimd_ShiftLeftLogicalAndInsertScalar:
+        case NI_AdvSimd_ShiftRightAndInsert:
+        case NI_AdvSimd_ShiftRightLogicalAndInsertScalar:
+            immLowerBound = 1;
```

Comment:
`immLowerBound == 0` is a `MOVI` variant? (Trying to make sure I'm reading the architecture manual correctly here 😄)

Reply:
yeah `0` would be a `MOV` or `ORR` (not `MOVI` since that's a bitmask).